todo_app/mongo_service.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+from typing import List
+from bson.objectid import ObjectId
+from pymongo.results import InsertOneResult, UpdateResult
+from todo_app.task import Task
+import pymongo
+from dotenv import load_dotenv
+from pymongo.mongo_client import MongoClient
+import os
+
+class MongoService:
+    def __init__(self, db_name: str):
+        client = self._connect_to_client(db_name)
+        self.task_collection = client[db_name].tasks
+
+    @staticmethod
+    def _connect_to_client(db_name: str) -> MongoClient: 
+        load_dotenv()
+        user = os.environ.get("DB_USER")
+        password = os.environ.get("DB_USER_PASSWORD")
+        cluster_url = os.environ.get("CLUSTER_URL")
+        return pymongo.MongoClient(f"mongodb+srv://{user}:{password}@{cluster_url}/{db_name}?ssl=true&retryWrites=true&w=majority")
```

Comment:
Minor, but it might be easier to just collect all of these settings together and have one environment variable for the connection string;
```python
connection_string = os.environ.get("DB_CONNECTION_STRING")
return pymongo.MongoClient(connection_string)
```